### PR TITLE
abc-clone does not update existing repos by default

### LINKED ISSUE #373: abc-clone - skip existing should have a default value of False
this is an easy fix - update the doc string and then the function. it works well in the notebook!!

### PR DESCRIPTION
**Pull request by kcranston**
Closes #373

### CODE REVIEW

File: abcclassroom/clone.py [hunk 1]
```diff
@@ -12,11 +12,11 @@
 from . import github as gh
 
 
-def clone_or_update_repo(organization, repo, clone_dir, skip_existing):
+def clone_or_update_repo(organization, repo, clone_dir, update_existing):
```
Comment (lwasser) [inline abcclassroom/clone.py:15]: thank you @kcranston !! so when i tried to run this in the notebook it was telling me i had to specify a value for update_existing (skip_existing). We don't have a default value for it as it stands. i can play with this some more but i was thinking we should have to specify update_existing and have some default value which you can overwrite if you wish? 

this all looks great however but that was the issue that i had.

Comment (kcranston) [reply abcclassroom/clone.py:15]: You don't happen to have the exact error, do you?

Comment (lwasser) [reply abcclassroom/clone.py:15]: I just tested it -  i updated the hub last night to 0.1.9 and the error is now gone!!

i have a notebook  on the hub that tests the notebook version of using abc-classroom!! so you can ignore my comment.

**Comment by kcranston** [comment]
I think we can close this without merge, as the actual issue was a bug in the git pull method, not that we wanted a change in the default behaviour for `abc-clone`. See PR #379

**Pull request closed without merging**